src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx

```diff
@@ -193,12 +193,10 @@ All parameters are case-insensitive.
     in the command, because all characters typed after the File parameter name are interpreted
     as the script file path followed by the script parameters.
 
-    Example: powershell HelloWorld.ps1
+    Example: pwsh HelloWorld.ps1
 
 -Help | -h | -? | /?
-    Shows this message. If you are typing a 'powershell' command in PowerShell on Windows
-    system, prepend the command parameters with a hyphen (-), not a forward slash (/).
-    You can use either a hyphen or forward slash in Cmd.exe.
```

Comment:
In case you wonder why this is removed. I tried in windows powershell and pwsh on windows, both hyphen (-) and forward slash (/) work.